codeforces/src/main/java/org/example/jumpGame/JumpGame.java

```diff
@@ -0,0 +1,69 @@
+package org.example.jumpGame;
+
+import java.util.List;
+
+public class JumpGame {
+    public int minimumJumps(List<Integer> listToJump) {
+
+        if (!isJumpPossible(listToJump)) {
+            throw new IllegalStateException("Jump is not possible.");
+        }
+
+        int currentIndex = 0;
+        int jumpsToReturn = 0;
+        int longestPossibleJump;
+        int placeInArray;
+        int difference = 0;
+
+        while (listToJump.get(currentIndex) + currentIndex < listToJump.size() - 1) {
```

Comment:
zwin to do funkcji bo bardzo skomplikowany jest ten warunek, az sie czytac tego nie chce :D